src/ray/raylet/scheduling/cluster_resource_manager.cc

```diff
@@ -250,14 +250,19 @@ bool ClusterResourceManager::UpdateNodeNormalTaskResources(
           /*requires_object_store_memory=*/false);
       auto &local_normal_task_resources = node_resources->normal_task_resources;
       if (normal_task_resources != local_normal_task_resources) {
+        // Return the current normal task resources back to the available resources.
+        AddNodeAvailableResources(node_id, local_normal_task_resources);
```

Comment:
I'm worried about resource overflow or underflow If you directly merge the resources of normal task into node resources.

Reply:
Yes, there are potential risks. I'll temporarily revert this part.